src/apps/AppFactory.cpp

```diff
@@ -10,6 +10,7 @@
 #include "scATTRACT.h"
```

Comment:
I think I already refactored this AppFactory. resolve your conflicts first and then make a pull request.